### PR TITLE
fix: few more external links open in new tabs

### PR DESCRIPTION
**Pull request by codedstrings**
I didn't change the links in the page footer cause I felt its not technically an external link..

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -265,10 +265,10 @@ <h4 class="my-3">Ventures</h4>
               <img class="mx-auto rounded-circle" src="assets/img/team/current/principal.jpg" alt="..." />
               <h4>Dr. S Brilly Sangeetha</h4>
               <p class="text-muted">Principal, IESCE</p>
-              <a class="btn btn-dark btn-social mx-2" href="https://linkedin.com/in/brilly-sangeetha-5b6a6a122/"><i
+              <a class="btn btn-dark btn-social mx-2" href="https://linkedin.com/in/brilly-sangeetha-5b6a6a122/" rel="noopener noreferrer" target="_blank"><i
```
Comment (nandgator) [inline index.html]: LinkedIn id changed, its [`https://www.linkedin.com/in/dr-s-brilly-sangeetha-5b6a6a122/`](https://www.linkedin.com/in/dr-s-brilly-sangeetha-5b6a6a122/) now.

**Review by nandgator — CHANGES_REQUESTED** [review]
Everything else's good. Just add one more commit to change that link and push on test branch itself.. nothing more to do.

File: index.html (hunk 1 above)
Comment (codedstrings) [reply index.html]: changes added

**Review by nandgator — APPROVED** [review]
LGTM

**Pull request merged**